docs/resources/secondary_database.md

```diff
@@ -0,0 +1,82 @@
+---
+page_title: "snowflake_secondary_database Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  A secondary database creates a replica of an existing primary database (i.e. a secondary database). For more information about database replication, see Introduction to database replication across multiple accounts https://docs.snowflake.com/en/user-guide/db-replication-intro.
+---
+
+# snowflake_secondary_database (Resource)
+
+A secondary database creates a replica of an existing primary database (i.e. a secondary database). For more information about database replication, see [Introduction to database replication across multiple accounts](https://docs.snowflake.com/en/user-guide/db-replication-intro).
+
+## Example Usage
+
+```terraform
+resource "snowflake_secondary_database" "test" {
+  name          = "database_name"
+  as_replica_of = "organization_name.account_name.primary_database_name"
+  is_transient  = false
+
+  data_retention_time_in_days {
+    value = 10
+  }
+
+  max_data_extension_time_in_days {
+    value = 20
+  }
+
+  external_volume       = "external_volume_name"
+  catalog               = "catalog_name"
+  default_ddl_collation = "en_US"
```

Comment:
What about REPLACE_INVALID_CHARACTERS and STORAGE_SERIALIZATION_POLICY? In other places as well.

Reply:
lol... It must've been added today or late Friday because it wasn't there. They're only useful for Iceberg tables, but I'll add them.